default/python/tests/README.md

```diff
@@ -1,15 +1,20 @@
 # Test runner
 
-This test are supposed to be run outside the game engine.
+These tests are supposed to be run outside the game engine.
+Tests run automatically on Travis for each PR
+and can be executed on the local machine.
 
-## Requirements
- - install [pytest](http://pytest.org/latest/getting-started.html)
+## Requirements for local run
+ - Install [pytest](http://pytest.org/latest/getting-started.html),
+   use the same version as mentioned in [.travis.yml](/.travis.yml#L1).
```

Comment:
I think the "#L1" in your reference here, which causes the first line to be highlighted, is not really helpful and perhaps a bit distracting.  Since the file is over a hundred lines long I think I'd also give them a slightly more specific pointer, perhaps something like 
```
use the version listed in the "pip install pytest==" line of [.travis.yml](/.travis.yml).
```